test/unit/computed/channel.spec.js

```diff
@@ -49,6 +52,12 @@ describe('Computed Channels Unit Tests', () => {
       expect(store.showChannelAlert, 'to equal', true);
     });
 
+    it('should not show channel alert after navigating to view', () => {
+      nav.goChannels();
```

Comment:
Isn't the flag set to false by the channel action `init()`?

Reply:
good point, fixed